flink-clients/src/main/java/org/apache/flink/client/cli/CliFrontendParser.java

```diff
@@ -241,8 +255,11 @@ private static Options getSavepointOptionsWithoutDeprecatedOptions(Options optio
 	 * Prints the help for the client.
 	 */
 	public static void printHelp() {
-		System.out.println("./flink <ACTION> [OPTIONS] [ARGUMENTS]");
+		System.out.println("./flink [CONFIGDIR] <ACTION> [ACTION-OPTIONS] [ARGUMENTS]");
```

Comment:
Shouldn't this be `./flink [--configDir <configDir>] <ACTION> [ACTION-OPTIONS] [ARGUMENTS]`?